common.mk

```diff
@@ -1,3 +1,4 @@
+GOPATH=$(shell eval $$(go env | grep GOPATH) && echo $$GOPATH)
```

Comment:
this is ... 

could you add a root make target (say something install-vendor depends upon) which checks if GOPATH is a valid directory